test/functional/fixtures/shell-test.c

```diff
@@ -71,8 +70,12 @@ int main(int argc, char **argv)
         fprintf(stderr, "Not enough REP arguments\n");
         return 4;
       }
-      uint8_t number = (uint8_t) *argv[2];
-      for (uint8_t i = 0; i < number; i++) {
+      int count = 0;
+      if (sscanf(argv[2], "%d", &count) != 1) {
```

Comment:
while you're in the area, PVS had this complaint about the other scanf() in this file :)

```
./test/functional/fixtures/shell-test.c:107:1: warning: V576 Incorrect format. Consider checking the third actual argument of the 'sscanf' function. It's dangerous to use string specifier without width specification. Buffer overflow is possible.
```

Reply:
Done.

Reply:
JFI: `cppcheck` also warns about it:

> [test/functional/fixtures/shell-test.c:107]: (warning) sscanf() without field width limits can crash with huge input data.